github-issues.el

```diff
@@ -225,8 +225,8 @@
     (message "No current issue selected")))
 
 (defvar github-issues-mode-map
-  (let ((map (make-keymap)))
-    (define-key map "\C-cr" 'github-issues-refresh)
+  (let ((map (make-sparse-keymap)))
+    (define-key map "g" 'github-issues-refresh)
```

Comment:
`make-keymap` returns a keymap which has empty entries for all keys. `make-sparse-keymap` returns an empty keymap. That's almost always what you want.